sdk/core/azure_core/README.md

```diff
@@ -201,6 +201,20 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 }
 ```
 
+## Performance Testing
```

Comment:
Package READMEs are for devs using our packages. This is of no interest to them and should not be here. It's more for people developing our crates, in which case the root `CONTRIBUTING.md` is sufficient.